src/Pages/ProductLists/Components/CategoryItems.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component, Fragment } from "react";
+import { APIYJ } from "../../../config";
```

Comment:
굿!
그런데 띄워쓰는 부분에는 밑줄 넣을 수 있도록 변수명 수정해주세요!
```suggestion
import { API_YJ } from "../../../config";
```

Reply:
넵 수정하겠습니다!